sympy/polys/factortools.py

```diff
@@ -124,13 +124,29 @@ def dmp_trial_division(f, factors, u, K):
 
 
 def dup_zz_mignotte_bound(f, K):
-    """Mignotte bound for univariate polynomials in `K[x]`. """
-    a = dup_max_norm(f, K)
-    b = abs(dup_LC(f, K))
-    n = dup_degree(f)
-
-    return K.sqrt(K(n + 1))*2**n*a*b
+    """
+    The Knuth-Cohen variant of Mignotte bound for
+    univariate polynomials in `K[x]`.
 
+    Reference: John Abbott, Journal of Symbolic Computation 50 (2013) 532–563.
+    """
+    d = dup_degree(f)
+    delta = _ceil( d / 2 )
+    
```

Comment:
In this line you've a trailing whitespace(i.e the line is not blank it has few spaces).